signalfx-org-metrics/metrics.yaml

```diff
@@ -1799,3 +1799,167 @@ sf.org.ui.num.pagevisits:
+    * Data resolution: 10 seconds
+ metric_type: Counter
+ title: sf.org.rum.numAddSpansCalls
+
+sf.org.rum.numSpansDroppedInvalid:
+  brief: Number of spans dropped because they were invalid. 
+  description: |
+    Number of spans dropped because they were invalid. Some of the reasons that spans are invalid are: spans are too large, missing required tags and invalid Trace IDs. Look through the reason column in the data table to see details for specific spans. 
+
+    * Dimension: orgId, reason, 
+    * Data resolution: 10 seconds
+  metric_type: Counter
+  title: sf.org.rum.numSpansDroppedInvalid
+
+sf.org.rum.numSpansDroppedInvalidByToken:
+  brief: The number of invalid spans Splunk RUM receives for a specific access
+    token.
+  description: | 
+    The number of invalid spans Splunk RUM receives for a specific
+    access token. Some of the reasons that spans are invalid are: spans are too large, missing required tags and invalid Trace IDs. Look through the reason column in the data table to see details for specific spans. 
+
+    * Dimensions: orgId, tokenId, reason
+    * Data resolution: 10 seconds
+
+  metric_type: Counter
+  title: sf.org.rum.numSpansDroppedInvalidByToken
+
+sf.org.rum.numSpansDroppedThrottle:
+  brief: The number of spans Splunk RUM dropped after you exceeded the allowed ingest volume.
+  description: |
+    The number of spans Splunk RUM dropped after you exceeded the allowed ingest volume.
+
+    * Dimension: orgId
+    * Data resolution: 10 seconds
+  metric_type: Counter
+  title: sf.org.rum.numSpansDroppedThrottle
+
+sf.org.rum.numSpansDroppedThrottleByToken:
+  brief: The number of spans Splunk RUM dropped after you exceeded the allowed ingest volume for a specific access token. 
+  description: |
+    The number of spans Splunk RUM dropped after you exceeded the allowed ingest volume for a specific access token. 
+
+    * Dimension: orgId, tokenId
+    * Data resolution: 10 seconds
+  metric_type: Counter
+  title: sf.org.rum.numSpansDroppedThrottleByToken
+
+
+
+sf.org.rum.numSpanBytesReceived:
+  brief:  The bytes of all the spans accepted and counted by the metric numSpansReceived.
+  description: |
+    The bytes of all the spans accepted and counted by the metric numSpansReceived.
+
+    * Dimensions: orgId, tokenId
+    * Data resolution: 10 seconds
+  metric_type: Counter
+  title: sf.org.rum.numSpanBytesReceived
+
+sf.org.rum.numSpanBytesReceivedByToken:
+  brief: The bytes of all the spans accepted and counted by the metric numSpansReceived by token. 
+  description: |
+    The bytes of all the spans accepted and counted by the metric numSpansReceived by token. 
+
+    * Dimensions: orgId, tokenId
+    * Data resolution: 10 seconds
+  metric_type: Counter
+  title: sf.org.rum.numSpanBytesReceivedByToken
+
+
+sf.org.rum.grossSpanBytesReceivedByToken:
+    brief: The bytes (uncompressed) of all the spans received and counted by the metric grossSpansReceived before throttling or filtering by token. 
```

Comment:
Why does it have a different data resolution from grossSpansReceived, out of curiosity?

Reply:
hi!  I would ask Matt Pound :)

Reply:
@mdubbyap hey Matt, would you please review those metrics and approve the PR if all looks good? Thanks!